website/content/docs/upgrading/upgrade-specific.mdx

```diff
@@ -29,6 +29,12 @@ earlier. If you operate Consul service mesh using Nomad 1.4.2 or earlier, do not
 [hashicorp/nomad#15266](https://github.com/hashicorp/nomad/issues/15266) is
 fixed.
 
+For 1.14.0, there is a known issue with `consul connect envoy`. If the command is configured
+to utilize TLS for contacting the HTTP API, it will also incorrectly enable TLS for gRPC.
```

Comment:
```suggestion
to use TLS for contacting the HTTP API, it will also incorrectly enable TLS for gRPC.
```
I think this one is in a style guide somewhere.